### ISSUE TITLE
Focal loss

### ISSUE THREAD

**Issue by ilyanelken** [Stick-To/CenterNet-tensorflow#28]
Hi,

I see that in line 249 there is:
 keypoints_neg_loss = -tf.pow(1.-reduction, 4) * tf.pow(tf.sigmoid(keypoints), 2.) * (-keypoints+tf.log_sigmoid(keypoints)) * (1.-gt_keypoints)

My question is about the following part:
**(-keypoints+tf.log_sigmoid(keypoints))**

Should not it be:
log(1 - sigmoid(keypoints)
?

Thanks,
Ilya

**Comment by Stick-To** [comment]
they are same , see https://www.tensorflow.org/api_docs/python/tf/math/log_sigmoid

**Comment by ilyanelken** [comment]
@Stick-To Right, thanks!
